packages/core/src/attestation/Attestation.ts

```diff
@@ -140,9 +140,15 @@ export function verifyAgainstCredential(
   attestation: IAttestation,
   credential: ICredential
 ): boolean {
```

Comment:
It would be more consistent though, if all the verify functions do the same, e.g. they all throw.

Reply:
I looked into it and you're right, this is a larger issue. We still have inconsistent behavior with regards to throw vs return boolean, to much to simply fix this here. I reverted the commit and pulled it into a separate PR: https://github.com/KILTprotocol/sdk-js/pull/632